src/components/UserBox.vue

```diff
@@ -286,6 +287,30 @@ export default {
             this.network.ircClient.raw('KICK', this.buffer.name, this.user.nick, reason);
         },
         createBanMask: function banMask() {
+            // if the account name is in the host ban the host
+            let accTest = new RegExp('^(.*' + this.user.account + '.*)$');
```

Comment:
While it's unlikely to be a problem, it's probably good practice to `_.escapeRegExp` the account when building the regex.